fooof/objs/utils.py

```diff
@@ -116,6 +113,41 @@ def average_fg(fg, bands, avg_method='mean', regenerate=True):
     return fm
 
 
+def average_reconstructions(fg, avg_method='mean'):
+    """Average across model reconstructions for a group of power spectra.
+
+    Parameters
+    ----------
+    fg : FOOOFGroup
+        Object with model fit results to average across.
+    avg : {'mean', 'median'}
+        Averaging function to use.
+
+    Returns
+    -------
+    freqs : 1d array
+        Frequency values for the average model reconstruction.
+    avg_model : 1d array
+        Power values for the average model reconstruction.
+        Note that power values are in log10 space.
+    """
+
+    if not fg.has_model:
+        raise NoModelError("No model fit results are available, can not proceed.")
+
+    avg_funcs = {'mean' : np.nanmean, 'median' : np.nanmedian}
+    if avg_method not in avg_funcs.keys():
+        raise ValueError("Requested average method not understood.")
```

Comment:
The above checks are duplicated in average_fg and could be moved to a shared check func.

Reply:
Yeh, I agree! I didn't go for that here, because there are related changes in #283 that add a general function for averaging data, but I didn't bother trying to backport that, and so the plan is to consolidate this into the 2.0 release and clean up the code here when merging!